cogs/fun/app_commands/apis.py

```diff
@@ -0,0 +1,33 @@
+import discord
+from discord import app_commands
+
+import random
+
+from utils import DuckCog
+
```

Comment:
Reorder the imports to follow the [default] -> [external modules] -> [local imports] rule

```suggestion
import random

import discord
from discord import app_commands

from utils import DuckCog
```